### ISSUE TITLE
Function Execution from Azure Portal not working

### ISSUE THREAD

**Issue by Sebs030** [thdeltei/azure-function-r#3]
I created a very simple R function which is just printing a random number scatterplot. I have setup the function in as described in this example, however if I want to run this function from the portal by clicking the "Run" button I get the following response in the logfile:

2017-12-12T15:59:13.626 Function started (Id=5686a706-3484-4045-a4b2-fee28ba95ece)
2017-12-12T15:59:18.830 Function completed (Success, Id=5686a706-3484-4045-a4b2-fee28ba95ece, Duration=5206ms)

However nothing really happens, i.e. the function doesn´t execute and no output/result is generated. 
If I run the same run.ps1 in the kudu powershell directly everything works as expected and i get the desired result.
Any idea what could be the issue? Thanks

**Comment by MarkusLund** [comment]
@Sebs030 did you get this working? I am having the same issue.

**Comment by Sebs030** [comment]
No, I had contacted Azure support and they replied that R is not an officially supported language, therefore issues like these can occur

**Comment by Sebs030** [comment]
Another option we will test is now to deploy R in a docker container in an Azure App Service - that would be somewhat severless then as well - let´s see

**Comment by MarkusLund** [comment]
Ok, thank you for answering.
